frontend-react/package.json

```diff
@@ -9,10 +9,12 @@
         "@okta/okta-react": "^6.4.1",
         "@okta/okta-signin-widget": "^6.0.1",
         "@rest-hooks/rest": "^3.0.3",
+        "@tanstack/react-query": "^4.2.1",
```

Comment:
Did a sanity check on this new 3rd party library. Seems legit. Lots of contributors/releases.